### ISSUE TITLE
packaging/feature request: Helm chart repository

### ISSUE THREAD

**Issue by marshallford** [opendistro-for-elasticsearch/community#156]
It would be nice if the Helm chart for opendistro was hosted on a [Helm chart repository](https://helm.sh/docs/topics/chart_repository/). Ideally the existing chart would be migrated to the [default stable repo](https://github.com/helm/charts/tree/master/stable) for easy consumption but a project specific repo would be a step in the right direction as well. Check out [Helm Hub](https://hub.helm.sh/charts) for a list of companies/communities running their own repos.

Thank you!

**Comment by DandyDeveloper** [comment]
@marshallford I agree with this, https://hub.helm.sh/ is absolutely where this should go.

**Comment by krishofmans** [comment]
We would also like to see a helm repo for this chart instead of just somewhere in a repo.

**Comment by marshallford** [comment]
@krishofmans exactly, the first step (before Helm Hub) is adding the chart to a new or existing repo **and then** adding the repo the Helm Hub list.

**Comment by DandyDeveloper** [comment]
@marshallford We really need Amazon owners of this repo to acknowledge a few things first, we also have a situation as I've described here: https://github.com/opendistro-for-elasticsearch/community/issues/157

I am frequently making improvements to this chart but it's currently in two opendistro repositories...

**Comment by alolita** [comment]
The stable version for the Open Distro Helm chart is in opendistro-build 

Please do not file PRs on the community repo version. We'll also take a look at hub.helm.sh :-) Thanks for the suggestion!

**Comment by krishofmans** [comment]
Yes, but still, the issue remains, it's only available on *GIT* repositories, which is not the way helm charts are meant to be distributed, they need to be in a convenient form for consumers as a *HELM* repository.

https://opendistro.github.io/for-elasticsearch-docs/docs/install/helm/
this:
![image](https://user-images.githubusercontent.com/999774/75014633-0afea480-5487-11ea-8384-5fd7e3d3de34.png)

Is a very inconvenient form of installation where you would typically just add the helm repo and be able to run the install command:
For example keycloak can be installed easily like this: https://hub.helm.sh/charts/gabibbo97/keycloak-gatekeeper
![image](https://user-images.githubusercontent.com/999774/75014706-36818f00-5487-11ea-9e1d-5a189cd9fff8.png)

**Comment by alolita** [comment]
@krishofmans Thanks for everyone's feedback on this thread. We'll take a look at possible alternatives for easier distribution of the Helm chart. Leaving this thread open for now.

**Comment by webwurst** [comment]
Also, it would be good to remove the chart in this repo if it's recommended to use the one from `opendistro-build`. Every now and then people provide fixes here. Those don't get merged, the chart looks outdated and this conveys the feeling no one cares. Just a link here to `opendistro-build` would help with that :)

**Comment by samarthya** [comment]
> Yes, but still, the issue remains, it's only available on _GIT_ repositories, which is not the way helm charts are meant to be distributed, they need to be in a convenient form for consumers as a _HELM_ repository.


On which version of helm and K8S did it work for you? I am trying to deploy 1.8.0 did not work for me.
